pytorch_lightning/core/memory.py

```diff
@@ -180,6 +182,7 @@ def __init__(self, model, mode: str = MODE_DEFAULT):
         self._model = model
         self._mode = mode
         self._layer_summary = self.summarize()
+        self._precision_megabytes = (self._model.precision / 8.0) * 1e-6 # 1 byte -> 8 bits)
```

Comment:
why 1e-6 and not 1/1024**2??

Reply:
@awaelchli pointed out good point
1 Mb = 1000**2
1 MiB = 1024 ** 2
as we are using MB hence 1e6

Reply:
Oh TIL :)